atd-vze/src/Components/Notes/Notes.js

```diff
@@ -10,17 +10,18 @@ import {
   Button,
 } from "reactstrap";
 import moment from "moment";
-import { notesDataMap } from "./notesDataMap";
-import {
+import { notesDataMap } from "./notesDataMap.js";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare a notes component
+const Notes = ({
+  crashLocationId,
```

Comment:
Just thinking out loud here - this id name could be updated to be more general like `recordId` or something - not a huge deal at all. Naming is always tricky, and I am no authority on it!

Reply:
Yeah coming up with that variable name was tricky, I like your idea of making it more general!